question-bank.json

```diff
@@ -193,7 +193,7 @@
             "How many lawyers are there in the United States?": 6,
             "How much carbon dioxide in grams is emitted annually into the atmosphere by all U.S. power plants?": 15,
             "Intel factories produce enough transistors per year for each man, woman, and child on earth to have 20 million. How many transistors are being produced per year?": 17,
-            "How many milliliters of water are in the Great Lakes": 18,
+            "How many milliliters of water are in the Great Lakes": 19,
```

Comment:
Thanks for the pull! Do you have a source or some math to back up the change? Thanks again.

Reply:
The volume of the Great Lakes is 2.3 * 10^16 Liters which is 2.3 * 10^19 milliliters